### PR TITLE
Orbit determination doppler

### PR DESCRIPTION
**Pull request by apak-00**
GSOC2020 wrap up.

### CODE REVIEW

File: orbitdeterminator/doppler/utils/utils_aux.py [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+            
+    elif frame == 'teme':
+        # Need some workaround conversions for TEME frame
+        site = EarthLocation(lat=lat*u.deg, lon=lon*u.deg, height=height/1e3*u.km)
+        site_itrs_temp = site.get_itrs(obstime=obstime)
+
+        r_itrs = CartesianRepresentation(
+            site_itrs_temp.data.xyz.value[0,:], 
+            site_itrs_temp.data.xyz.value[1,:], 
+            site_itrs_temp.data.xyz.value[2,:], unit=u.km)
```
Comment (astrojuanlu) [inline orbitdeterminator/doppler/utils/utils_aux.py:139]: I'd say that

```suggestion
        r_itrs = site_itrs_temp.cartesian
```

does the same (see other instances of this below).

**Comment by hornig** [comment]
Hi, nice work! well done.
how shall the code be accessed from outside? is it something we should still have as experimental and accessed via example files or do you think this should be accessed from a main function?

**Comment by apak-00** [comment]
> 
> 
> Hi, nice work! well done.
> how shall the code be accessed from outside? is it something we should still have as experimental and accessed via example files or do you think this should be accessed from a main function?

All of the orbit determination functions are documented and can be accessed individually. It is no longer a separate module, so it should also be good to merge (no conflicts).

**Comment by hornig** [comment]
@apak-00 
What I do not understand are these:

- there is an archive folder. is this still needed?
- there is a temp.py.
- and the readme is sort of empty.

What shall we do there?

**Pull request merged**